images/base/Dockerfile

```diff
@@ -43,20 +43,23 @@ USER ${NB_UID}
 # Set environment variable for Conda cache directory inside the container
 ENV CONDA_PKGS_DIRS=/opt/conda/pkgs
 
-RUN conda install -n base conda-libmamba-solver && \
-  conda config --set solver libmamba
+# Uninstall default R version
+RUN conda uninstall r-base && conda clean --all -y
```

Comment:
Could you merge the two RUN operations on this line and the next in order to minimize the size of all the layers please ?